MAINTAINERS.md

```diff
@@ -1,31 +1,29 @@
 | --------------------- | ---------------------------------------------------- | ------------------------------- | ----------------------------------------------------------------------------- |
 | Makoto Takemiya       | [@takemiyamakoto](https://github.com/takemiyamakoto) | takemiya@soramitsu.co.jp        | Product vision                                                                |
 | Dmitri Venger         | [@dmitrivenger](https://github.com/dmitrivenger)     | venger@soramitsu.co.jp          | Project manager                                                               |
 | Bogdan Mingela        | [@Mingela](https://github.com/Mingela)               | mingela@soramitsu.co.jp         | Iroha Team Lead                                                               |
-| Aleksandr Petrosyan   | [@appetrosyan](https://github.com/appetrosyan)       | petrosyan@soramitsu.co.jp       | Iroha 2 architect                                                             |
+| Aleksandr Petrosyan   | [@appetrosyan](https://github.com/appetrosyan)       | ap886@cantab.ac.uk              | Iroha 2 architect                                                             |
 | Daniil Polyakov       | [@Arjentix](https://github.com/Arjentix)             | daniil.polyakov@soramitsu.co.jp | Development: Rust                                                             |
-| Ilia Churin           | [@ilchu](https://github.com/ilchu)                   | churin@soramitsu.co.jp          | Development: Rust                                                             |
 | Marin Veršić          | [@mversic](https://github.com/mversic)               | versic@soramitsu.co.jp          | Tech lead, development: Rust, Java                                            |
 | Sam H. Smith          | [@SamHSmith](https://github.com/SamHSmith)           | smith@soramitsu.co.jp           | Development: Rust, C                                                          |
 | Shanin Roman          | [@Erigara](https://github.com/Erigara)               | shanin@soramitsu.co.jp          | Development: Rust                                                             |
-| Vladimir Pesterev     | [@pesterev](https://github.com/pesterev)             | pesterev@soramitsu.co.jp        | Development: Rust                                                             |
 | Dmitry Balashov       | [@0x009922](https://github.com/0x009922)             | dbalashov@soramitsu.co.jp       | Development: Rust, TypeScript, JavaScript                                     |
-| Grzegorz Bazior       | [@baziorek](https://github.com/baziorek)             | bazior@agh.edu.pl               | Development: C++, Python, JavaScript, TypeScript                              |
+| Grzegorz Bazior       | [@baziorek](https://github.com/baziorek)             | bazior@agh.edu.pl               | Development: C++, Python                             |
```

Comment:
one more small change. I learned Iroha 1, because I was working in some company, so I should use their mail:
```suggestion
| Grzegorz Bazior       | [@baziorek](https://github.com/baziorek)             | g.bazior@yodiss.pl               | Development: C++, Python                                                      |
```